### PR TITLE
Reservation instructions

### PR DESCRIPTION
**Pull request by kevinstratford**
For #639

Could someone check my interpretation is correct?

### CODE REVIEW

File: docs/user-guide/scheduler.md [hunk 1]
```diff
@@ -768,7 +768,7 @@ in partition standard
 You can use the `squeue` command to show the current estimated start time for a job.
 Please note that it is just an estimate, the actual start time may differ as the scheduler
 status when the start time was estimated may be different due to subsequent changes to
-the scheduler state. To return the estimated start time for a job you spacify the 
+the scheduler state. To return the estimated start time for a job you spacify the
```
Comment (aturner-epcc) [inline docs/user-guide/scheduler.md:771]: "spacify" -> "specify"

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Looks fine apart from a typo (which predates, this PR I think but it is a good opportunity to fix!)

File: docs/user-guide/scheduler.md (hunk 1 above)
Comment (kevinstratford) [reply docs/user-guide/scheduler.md]: Done.

**Review by clairbarrass — APPROVED** [review]
lgtm

**Pull request merged**